Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnSearch.hs

```diff
@@ -320,6 +322,7 @@ buildQuote requestId providerInfo now searchRequest QuoteInfo {..} = do
         quoteDetails = quoteDetails',
         merchantId = searchRequest.merchantId,
         merchantOperatingCityId = searchRequest.merchantOperatingCityId,
+        serviceTierName = Nothing,
```

Comment:
Why is `serviceTierName = Nothing` for Quote?

Reply:
Fixed.